esmvaltool/recipes/recipe_multimodel_products_wp5.yml

```diff
@@ -0,0 +1,80 @@
+###############################################################################
+# nameslist_multimodel_products.yml
+#
+# Description
+# Namelist to compute the ensemble mean anomaly and the ensemble
+# variance and agreement and plot the results as maps and time series.
+#
+# Authors
+# Nicolaur Manubens (BSC)
+# Alasdair Hunter (BSC)
+#
+#
+# Project
+# KNMI Magic
+#
+# Reference
+# Manubens, Nicolau, Louis-Philippe Caron, Alasdair Hunter, Omar Bellprat,
+# Eleftheria Exarchou, Neven S. Fučkar, Javier Garcia-Serrano et al.
+# "An R package for climate forecast verification." Environmental
+# Modelling & Software 103 (2018): 29-42.
+#
+#
+# This namelist is part of the ESMValTool
+###############################################################################
+---
+
+datasets:
+  - {dataset: bcc-csm1-1, type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1961,  end_year: 1990}
```

Comment:
This dataset/variable is not available in the CP4CDS data on Jasmin, this is a problem for the Magic project

Reply:
Sorry, @bouweandela, I did the request but I don't have access to Jasmin yet. Could you please said me which variable isn't at Jasmin? Currently, I've seen that the recipe is configure to work with pressure at sea level.
Thanks

Reply:
The one used in this recipe, there is no `tas` directory in this folder:
/group_workspaces/jasmin2/cp4cds1/data/c3s-cmip5/output1/BCC/bcc-csm1-1/historical/mon/atmos/Amon/r1i1p1/